src/utils.ts

```diff
@@ -487,8 +505,8 @@ export const getCombatStylesForCategory = (style: EquipmentCategory): PlayerComb
     case EquipmentCategory.CROSSBOW:
     case EquipmentCategory.THROWN:
       ret = [
-        { name: 'Accurate', type: 'ranged', stance: 'Accurate' },
         { name: 'Rapid', type: 'ranged', stance: 'Rapid' },
+        { name: 'Accurate', type: 'ranged', stance: 'Accurate' },
```

Comment:
Not a fan of this approach. Indeed it would make "Rapid" the default combat style when equipping the weapon, but re-arranging the order of these no longer matches the in-game interface, which is a strong desire of ours (especially to help less smart people find their way around the UI).

Maybe instead of doing it this way, we could add some kind of condition [near here](https://github.com/weirdgloop/osrs-dps-calc/blob/staging/src/state.tsx#L547) that sets the style to rapid by default when switching to a bow weapon.

Reply:
Sure 👍

Reply:
Fixed.